### PR TITLE
Kirti/feat cms frontend

### PR DESCRIPTION
**Pull request by kirti18patel**
- Add landing page for neocon (header, hero banner, image carousel, downloads, footer)
- Add gtm for tracking
- Add local and google fonts
- Add skip to main navigation

### CODE REVIEW

File: components/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+			path: '../public/fonts/gotham/Gotham-Book.woff2',
+			weight: '400',
+			style: 'normal',
+		},
+	],
+	variable: '--font-gotham',
+});
+
+export const Layout = ({ children, story }: StoryBlokLayoutProps) => {
+	const content = story ? story.content : children.props.story.content;
```
Comment (Shaderpixel) [inline components/Layout.tsx:29]: This is problematic and causing build issues. story.content can be undefined, children.props.story.content can be undefined. You need to account for when both of them are undefined and also whether Header and Footer should still be rendered if they aren't

File: components/StaticImage.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import Image from 'next/image';
+
+export const StaticImage = ({ blok }: StoryBlokComponentProps) => {
+	return (
+		<div {...storyblokEditable(blok)}>
+			<Image
+				src={blok.static_image.filename}
+				alt={blok.static_image.alt}
+				className="u-w-full"
+				// TODO: confirm width and height
```
Comment (Shaderpixel) [inline components/StaticImage.tsx:12]: remove comment, height shouldn't be 2000, please reference the actual image width and height and update values

File: config.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+const config = {
+	// TODO: Add Google Analytics ID
```
Comment (Shaderpixel) [inline config.ts:2]: Do we have this value?

Comment (kirti18patel) [reply config.ts:2]: I don't have this value.

File: components/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+						</div>
+						<div>
+							<nav className="u-hidden u-bg-gfg-tertiary u-px-5%  u-text-white lg:u-block 2xl:u-px-24">
+								<ul className="u-flex u-flex-wrap u-divide-x u-divide-gray-400/20 2xl:u-flex-nowrap [&>*:last-child]:!u-border-r [&>*:last-child]:!u-border-gray-400/20">
+									{navbar.map((link: LinkProps, i: number) => (
+										<li
+											key={link._uid}
+											className={i === 4 ? 'lg:u-ml-auto' : ''}
+										>
+											{/* TODO: add u-font-sansGotham */}
```
[the remark this answers is not in the document]
Comment (kirti18patel) [reply components/Header.tsx]: Implemented u-font-sansGotham

File: components/Layout.tsx (hunk 1 above)
Comment (kirti18patel) [reply components/Layout.tsx]: This issue is resolved by adding proper condition.

**Review by Shaderpixel — APPROVED** [review]
let's get this merged!

**Pull request merged**